From: Dennis Dalessandro <dennis.dalessandro@intel.com>
Subject: IB/rdmavt: Remove signal_supported and comments
Git-commit: 0765b01b8e2da50ad56f6544f935f5eaef1389f2
Patch-mainline: v4.6
References: bsc#973818, fate#319242
Acked-by: Jiri Bohac <jbohac@suse.cz>

Initially it was intended that rdmavt would support some signaling
between the underlying driver and itself. However this turned out to be
unnecessary for qib and hfi1. If we need to add something like this in
later to support another driver we should do it then. As of now this
essentially dead code so remove it.

Reviewed-by: Ira Weiny <ira.weiny@intel.com>
Reviewed-by: Jubin John <jubin.john@intel.com>
Signed-off-by: Dennis Dalessandro <dennis.dalessandro@intel.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
---
 include/rdma/rdma_vt.h |   18 ------------------
 1 file changed, 18 deletions(-)

--- a/include/rdma/rdma_vt.h
+++ b/include/rdma/rdma_vt.h
@@ -138,24 +138,6 @@ struct rvt_driver_params {
 	struct ib_device_attr props;
 
 	/*
-	 * Drivers will need to support a number of notifications to rvt in
-	 * accordance with certain events. This structure should contain a mask
-	 * of the supported events. Such events that the rvt may need to know
-	 * about include:
-	 * port errors
-	 * port active
-	 * lid change
-	 * sm change
-	 * client reregister
-	 * pkey change
-	 *
-	 * There may also be other events that the rvt layers needs to know
-	 * about this is not an exhaustive list. Some events though rvt does not
-	 * need to rely on the driver for such as completion queue error.
-	 */
-	 int rvt_signal_supported;
-
-	/*
 	 * Anything driver specific that is not covered by props
 	 * For instance special module parameters. Goes here.
 	 */
